javagen/src/main/java/com/azure/autorest/mapper/ChoiceMapper.java

```diff
@@ -50,15 +50,13 @@ public IType map(ChoiceSchema enumType) {
 
             List<ClientEnumValue> enumValues = new ArrayList<>();
             for (ChoiceValue enumValue : enumType.getChoices()) {
-                String enumName = enumValue.getValue();
```

Comment:
@jianghaolu Can we still keep the non-Fluent flag here?

The reason is that m4 sometimes do strange naming for enum value, e.g. `IPv4` -> `I_PV4`.

Reply:
Oh I see. Yes I'll make the change

Reply:
Actually then this PR is no longer needed